pkg/cloudprovider/types/types.go

```diff
@@ -81,6 +82,32 @@ type NetworkConfig struct {
 	PodCIDRs []string `json:"podCIDRs"` // PodCIDRs fields is used to choose IPv4, IPv6 or dual-stack modes.
 }
 
+type IPVersion int
+
+const (
+	IPv4 = iota
+	IPv6
+)
+
+func (n NetworkConfig) ContainsCIDR(version IPVersion) bool {
```

Comment:
Can you please move this to https://github.com/kubermatic/machine-controller/blob/master/pkg/cloudprovider/util/net.go